src/pages/api/access_codes/create.ts

```diff
@@ -59,6 +59,17 @@ export default withRouteSpec({
     })
   }
 
+  // TODO: check time bound access codes and allow duplicate pin codes with different time frames
+  const duplicate_access_code = req.db.access_codes.find(
+    (ac) => ac.code === code && ac.device_id === device_id,
+  )
+  if (duplicate_access_code == null) {
+    throw new HttpException(409, {
+      type: "duplicate_access_code",
+      message: `Cannot set duplicate access code ${code} named ${name}`,
+    })
+  }
+
```

Comment:
🐧 I don't know how strictly we need to emulate connect behavior, but I'm seeing here that we show the time range if the access code is time-bound

https://github.com/seamapi/seam-connect/blob/main/lib/api-helpers/validate-and-insert-access-code.ts#L350-L366

Reply:
https://github.com/seamapi/fake-seam-connect/pull/168/commits/13a41ff236b53caff1150174cc153bbfcbe749ef